index.js

```diff
@@ -1,3 +1,4 @@
 const Bot = require("./src/Bot");
 
+// noinspection JSIgnoredPromiseFromCall
 Bot.getInstance().start();
```

Comment:
Just adding a noinspection comment isn't a solution. if start() throws an error, it will result in an unhandled rejection, which doesn't cause a crash in current nodejs versions. This could result in your process continuing to run while the actual bot doesn't.